09-valve-monitor/firmware/zephyr/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Container for Developing Zephyr Applications
 
 # Build development environment
-# $ docker build --file Dockerfile.zephyr --tag west .
+# $ docker build --tag west .
```

Comment:
So this is official stuff from Zephyr? Can we provide a link(s) to wherever that official stuff lives, for future reference? Just as a comment at the top of this file. Same for devcontainer.json, if relevant.

Reply:
The link would be `https://blues.dev/zephyr`. Do you think it's good to have?

Reply:
Sure, doesn't hurt.